squire_core/src/state/boilerplate.rs

```diff
@@ -16,3 +16,14 @@ impl Debug for SessionCommand {
         }
     }
 }
+
+impl Debug for AccountCommand {
```

Comment:
Not necessary for this PR, but I wonder if we could use the `derive_more` crates macros to generate this.